frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts

```diff
@@ -123,6 +124,15 @@ describe('Test network type presets and options', () => {
   const wizard = new Wizard();
   const leakedResources = new Set<string>();
 
+  const vm = new VMBuilder()
```

Comment:
it is easier to use getBasicVMBuilder here instead

Reply:
use getBasicVMBuilder here now.